repos/fdbt-site/src/pages/api/pointToPointPeriodProduct.ts

```diff
@@ -22,8 +23,11 @@ export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
             productDurationUnits: durationUnits,
         };
 
+        const fareTypeAttribute = getSessionAttribute(req, FARE_TYPE_ATTRIBUTE);
```

Comment:
If we use getFareTypeFromFromAttributes , we dont need to check to see if its defined.

Reply:
we can't use the getFareTypeFromFromAttributes in this file as the fare type can be period or school service and for school service we need to check for the term duration.

Reply:
OK - Can we instead move this check to see if its a school ticket into a function, because we're checking it in a few different places and running the same code